tests/e2e-pw/utils/devtools.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { Page, expect } from '@playwright/test';
+
+const goToDevToolsSettings = async ( page: Page ) => {
+	await page.goto( 'wp-admin/admin.php?page=wcpaydev', {
+		waitUntil: 'load',
+	} );
+};
+
+const saveDevToolsSettings = async ( page: Page ) => {
+	await page.getByRole( 'button', { name: 'Save Changes' } ).click();
+	expect( page.getByText( /Settings saved/ ) ).toBeVisible();
+};
+
+const getIsCardTestingProtectionEnabled = async ( page: Page ) => {
+	return (
+		( await page
+			.locator( '#wcpaydev_force_card_testing_protection_on:checked' )
+			.count() ) === 1
+	);
+};
```

Comment:
If I'm not wrong we could simplify this to something like the following suggestion, using `getByLabel` in `toggleCardTestingProtection` too. We should try to avoid using `locator` as much as possible.

```suggestion
const getIsCardTestingProtectionEnabled = async ( page: Page ) =>
	page.getByLabel( 'Card testing mitigations enabled' ).isChecked();
```

Reply:
I thought I tried to use this approach first, but I must have done something wrong. I followed your approach and it worked nicely. Thanks for the suggestion and the heads up! I've updated it on [8f0a3b7](https://github.com/Automattic/woocommerce-payments/pull/10104/commits/8f0a3b73a7310128b65c6415a4a5ce4b6a6edbb5).